spec/test-suite.json

```diff
@@ -0,0 +1,71 @@
+{
+    "suite":
+    {
+        "zmanim":
+        [
+            {
+                "setup":
+                {
+                    "calculator":
+                    {
+                        "name": "SunTimes",
+                        "zmanimInformation":
+                        {
+                            "name": "The SunTimes calculator based on the USNO algorithm.",
+                            "url": "https://github.com/KosherJava/zmanim/blob/master/src/net/sourceforge/zmanim/util/SunTimesCalculator.java"
+                        },
+                        "information":
+                        {
+                            "name": "US Naval Observatory's Almanac for Computer algorithm",
+                            "url": "http://aa.usno.navy.mil/publications/docs/asa.php"
+                        }
+                    },
+                    "date": "2010-04-02",
+                    "location":
+                    {
+                        "name": "Lakewood, NJ",
+                        "lat": 40.09596,
+                        "long": -74.22213,
+                        "elevation": 0,
+                        "timeZone":
+                        {
+                            "offset": -5,
+                            "zoneInfoName": "America/New_York"
+                        }
+                    }
+                },
+                "tests":
+                [
+                    {
+                        "method": "sunset",
+                        "description": "Gets the exact sunset time",
+                        "extra":
+                        {
+                            "edgeCase": false,
+                            "information": ""
+                        },
+                        "expected":
+                        {
+                            "utc": "2010-04-02T12:22:03.710Z",
```

Comment:
Having two expected outputs seems redundant and is probably going cause confusion. I'd suggest settling on one format, UTC being totally fine. A test suite for zmanim formatting would be nice, but given platform-specific nature, it may even be out of the scope of this project. 

(For example, iOS has `NSDateFormatter` and friends. The web has `Date()` and the `moment` project.)

Reply:
Instead of UTC, we can have just "expected: value" so that it should depend on the tz offset?

Reply:
The @KosherJava implementation does not return an offset time, the only one that I'm aware of that does is my own.  I think sticking with UTC is the best option, since it's easy to get to regardless of the implementation output, and doesn't rely on any specialized TZ rules (whether DST should or should not be applied).